lib/autoload/kuroko2/workflow/engine.rb

```diff
@@ -164,6 +185,15 @@ def notify_long_elapsed_time_if_needed(token)
           Kuroko2.logger.info(message)
         end
       end
+
+      def sleep_for_each_retry(node, token)
+        if token.context['RETRY'].present? && token.context['RETRY'][node.path].present?
+          started_time = Time.current.to_i
+          while started_time + token.context['RETRY'][node.path]['sleep_time'] > Time.current.to_i
+            # sleep
```

Comment:
I have fixed it in `ea00b5a` too.
I changed while loop to sleep function, and decided to keep the log.